.storybook/preview.ts

```diff
@@ -0,0 +1,29 @@
+import { addDecorator, addParameters } from "@storybook/html";
+import centered from "@storybook/addon-centered/html";
+import theme from "./theme";
+import { titlelessDocsPage } from "./utils";
+
+addDecorator(centered);
+addParameters({
+  backgrounds: [{ name: "Light", value: "#f8f8f8", default: true }],
+  options: {
+    theme,
+    storySort: (a, b) => (a[1].kind === b[1].kind ? 0 : a[1].id.localeCompare(b[1].id, undefined, { numeric: true }))
```

Comment:
how does this work?

Reply:
That sorts the stories by type and then numerically/alphabetically

Reply:
I got it from the [Storybook doc](https://storybook.js.org/docs/configurations/options-parameter/#sorting-stories). I tested, but didn't look into it. However, after digging a little bit this morning, I [found](https://github.com/storybookjs/storybook/pull/9188/files) a simpler way:

```ts
storySort: {
  method: "alphabetical"
}
```

Reply:
@jcfranco we actually have to use this to get the storybook sorted with mdx files at top, followed by components and then app components.

```ts
storySort: (a, b) => {
      const sectionA = a[1].id.split("-")[0];
      const sectionB = b[1].id.split("-")[0];

      return sectionB.localeCompare(sectionA);
    },
```

Reply:
Got it. I'll tackle that in the port PR.